pom.xml

```diff
@@ -73,7 +73,7 @@
 
     <brave.version>5.6.9</brave.version>
     <dropwizard.version>1.3.13</dropwizard.version>
-    <kafka-client.version>2.1.1</kafka-client.version>
```

Comment:
Let’s leave the Kafka-client version as-is for now, I need to check for any breaking changes